# ExecuteTrajectory.action

# Goal
float64 traj_duration
float64 acc_duration
float64 total_time
int32 trajectory_len
float64 kp
float64 end_position_x
float64 end_position_y
float64 end_position_z
string cmd_interface
string ctrl
string traj_type
string s_type
---
# Result
bool success
string message
---
# Feedback
float64 position_error